grammars.yml

```diff
@@ -10,7 +10,7 @@ vendor/grammars/Agda.tmbundle:
 vendor/grammars/Alloy.tmbundle:
 - source.alloy
 vendor/grammars/Assembly-Syntax-Definition:
-- source.assembly.unix
+- source.x86
```

Comment:
There are a several changes in this file that look unintentional and unrelated to this pull request.

Reply:
Changes were done by the script. But I'll undo these and commit again.